src/styles/layouts/hp.scss

```diff
@@ -61,10 +71,8 @@
 
 .arrow-box:nth-child(odd) {
   min-height: 340px;
-  clip-path: polygon(100% 80%, 50% 100%, 0 80%, 0 0, 50% 20%, 100% 0);
```

Comment:
I'm not sure about removing entirely clipped boxes.
The homepage needs something to stand out and without those arrow design the HP feels a bit blander especially 
on mobile viewports.

### Comparison

|  Before | After | 
|--------|-------|
| <img width="394" alt="Screenshot 2022-12-17 at 11 49 11" src="https://user-images.githubusercontent.com/16583125/208238367-b85af802-67a3-4362-9c65-158075a6b46e.png"> |   <img width="410" alt="Screenshot 2022-12-17 at 11 49 33" src="https://user-images.githubusercontent.com/16583125/208238384-9af963ae-5c48-4025-9386-d3d41b78dcb1.png"> |

### Suggested alternatives

I've iterated a bit on your changes and I've come up with these 2 designs:

| 1st Idea | 2nd Idea |
|---------|----------|
| <img width="373" alt="Screenshot 2022-12-17 at 12 09 59" src="https://user-images.githubusercontent.com/16583125/208238824-a16f197f-2ee0-49e8-bebb-15bb89c96526.png"> |  <img width="395" alt="Screenshot 2022-12-17 at 12 18 07" src="https://user-images.githubusercontent.com/16583125/208239142-15b82847-3956-4883-97b9-a484cecb868f.png"> |


### 1st idea diff

```diff
diff --git a/src/styles/layouts/hp.scss b/src/styles/layouts/hp.scss
index e10bba5..5d05676 100644
--- a/src/styles/layouts/hp.scss
+++ b/src/styles/layouts/hp.scss
@@ -66,22 +66,18 @@ .logo-hero {
 }
 
 .arrow-box {
+  background-color: var(--bg-secondary);
+  color: var(--text-secondary);
+  min-height: 240px;
   padding: 20px 16px 0;
 }
 
-.arrow-box:nth-child(odd) {
-  min-height: 340px;
-  background-color: var(--bg-primary);
-  color: var(--text-primary);
-}
-
 .arrow-box:nth-child(even) {
-  min-height: 240px;
-  color: var(--text-secondary);
-}
-
-.arrow-box:first-child {
   background-color: var(--bg-primary);
+  color: var(--text-primary);
+  min-height: 340px;
+  padding-top: 40px;
+  clip-path: polygon(100% 80%, 50% 100%, 0 80%, 0 0, 50% 20%, 100% 0);
 }
 
 .contacts-cta {

```


### 2nd idea diff

```diff
diff --git a/src/styles/layouts/hp.scss b/src/styles/layouts/hp.scss
index e10bba5..6f1503f 100644
--- a/src/styles/layouts/hp.scss
+++ b/src/styles/layouts/hp.scss
@@ -66,22 +66,17 @@ .logo-hero {
 }
 
 .arrow-box {
+  background-color: var(--bg-secondary);
+  color: var(--text-secondary);
+  min-height: 240px;
   padding: 20px 16px 0;
 }
 
-.arrow-box:nth-child(odd) {
-  min-height: 340px;
-  background-color: var(--bg-primary);
-  color: var(--text-primary);
-}
-
 .arrow-box:nth-child(even) {
-  min-height: 240px;
-  color: var(--text-secondary);
-}
-
-.arrow-box:first-child {
   background-color: var(--bg-primary);
+  color: var(--text-primary);
+  min-height: 340px;
+  clip-path: polygon(0 20%, 100% 0, 100% 80%, 0 100%);
 }
 
 .contacts-cta {
```

I personally believe that the 1st idea is the best for the HP.
Let me know what you think :)

Reply:
The problem with the V clip is that I found it quite "heavy" especially on the desktop viewports.

I like the 2nd idea, let's try it!

Reply:
Pushed the 2nd idea, looks nice!

Reply:
Neat!

Btw the Netlify PR preview helps a lot, thanks 👍.